cmake/BundledRocksDB.cmake

```diff
@@ -42,9 +42,9 @@ function(use_bundled_rocksdb SOURCE_DIR BINARY_DIR)
     if(PORTABLE)
         list(APPEND ROCKSDB_CMAKE_ARGS -DPORTABLE=ON)
     endif()
-	if(WIN32)
-		list(APPEND ROCKSDB_CMAKE_ARGS -DROCKSDB_INSTALL_ON_WINDOWS=ON)
-	endif()
+  if(WIN32)
```

Comment:
This is messed up, too. I'm assuming your tool only replaced tabs with 2 spaces, but not existing 4 space indentation with 2 spaces.